wormhole-connect/src/components/Options.tsx

```diff
@@ -47,15 +47,11 @@ type Option = {
 type Props = {
   children: Option[];
   active?: number;
+  onSelect: (key: any) => void;
 };
 
 function Options(props: Props) {
   const { classes } = useStyles();
-  // dispatch selectOption event
-  const emitSelectOption = (key: any) => {
-    const event = new CustomEvent('selectOption', { detail: key });
```

Comment:
There was something listening to this event in `GasOptions`, did you check that it isn't affected by this deletion?